python/pyspark/sql/tests/connect/test_connect_column_expressions.py

```diff
@@ -134,6 +134,16 @@ def test_list_to_literal(self):
         lit_list_plan = fun.lit([fun.lit(10), fun.lit("str")]).to_plan(None)
         self.assertIsNotNone(lit_list_plan)
 
+    def test_column_alias(self) -> None:
+        # SPARK-40809 - Support for Column Aliases
```

Comment:
nit:
```suggestion
        # SPARK-40809: Support for Column Aliases
```
per https://spark.apache.org/contributing.html